site/docs/pages/blocked.tsx

```diff
@@ -0,0 +1,8 @@
+export default function Login() {
+  return (
+    <div>
+      <h1>You have been Blocked</h1>
```

Comment:
Do we have a more correct message? `Blocked` seems a bit affermative, can you check with other teams what kind of messaging they use.

Reply:
Yes! This is 100% just for locally testing to make sure the redirect is working properly for me!